otaclient/ota_proxy/orm.py

```diff
@@ -133,6 +154,7 @@ def check_type(self, value: Any) -> bool:
         return self.type_checker(value)
 
 
+@dataclass_transform()
 class ORMeta(type):
     """This metaclass is for generating customized <TableCls>."""
```

Comment:
With `dataclass_transform`, IDE now can correctly identify the `ORMBase` derived subclass as dataclass.